### PR TITLE
SAP HANA metrics

### PR DESCRIPTION
**Pull request by dehaansa**
This draws in support for the SAP HANA receiver developed in open telemetry.

### CODE REVIEW

File: apps/saphana.go
Comment (ridwanmsharif) [inline apps/saphana.go:52]: Is this not supposed to be configurable? The metadata file says there will be an `insecure` config option?

**Review by ridwanmsharif — APPROVED** [review]
LGTM modulo a comment

**Comment by dehaansa** [comment]
All kokoro passed in https://github.com/GoogleCloudPlatform/ops-agent/pull/662/commits/b1da6bc9cdae83e1a73aef5f60497819005ec544 , not running again until more things are merged/changes are requested.

**Pull request merged**
